spec/features/multiple_emails/add_email_spec.rb

```diff
@@ -112,7 +112,9 @@
     sign_in_and_2fa_user(user)
 
     visit add_email_path
+    puts current_path
     expect(page).to have_current_path(account_path)
+    expect(page.has_content? t('users.emails.error_message'))
```

Comment:
Minor: Could use the chainable version of the same matcher, which is a bit more common:

```suggestion
    expect(page).to have_content t('users.emails.error_message')
```